core/flyout.js

```diff
@@ -1388,3 +1229,12 @@ Blockly.Flyout.prototype.reflow = function() {
     this.workspace_.addChangeListener(this.reflowWrapper_);
   }
 };
+
+/**
+ * @return {boolean} True if this flyout may be scrolled with a scrollbar or by
+ *     dragging.
+ * @package
+ */
+Blockly.Flyout.prototype.isScrollable = function() {
```

Comment:
Scrollbar isVisible is really isHandleVisible.  A flyout category always has a scrollbar, but if there's enough space that it doesn't need to scroll, the scrollbar hides itself and isVisible returns false.